README.md

```diff
@@ -1,3 +1,5 @@
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg)](LICENSE.txt) [![CircleCI](https://img.shields.io/circleci/project/github/RedSparr0w/node-csgo-parser.svg)](https://circleci.com/gh/vintasoftware/django-react-boilerplate)
```

Comment:
Those links don't look right, how did you generate them? Also we already have a badge for CircleCI at the `Project bootstrap` section.